index.html

```diff
@@ -296,14 +169,16 @@ <h4>Ali Sahid</h4>
       <section class="partners" id="partners">
         <h3>Partners</h3>
         <div class="border"></div>
-        <div class="box-partner-A">
-          <img src="./assets/images/airbnb.svg" alt="Airbnb" />
-          <img src="./assets/images/google.svg" alt="Google" />
-          <img src="./assets/images/mozilla.svg" alt="mozilla" />
-        </div>
-        <div class="box-partner-B">
-          <img src="./assets/images/daum.svg" alt="daumkakao" />
-          <img src="./assets/images/naver.svg" alt="naver" />
+        <div class="row">
+          <div class="box-partner-A">
+            <img src="./assets/images/airbnb.svg" alt="Airbnb" />
+            <img src="./assets/images/google.svg" alt="Google" />
+            <img src="./assets/images/mozilla.svg" alt="mozilla" />
+          </div>
+          <div class="box-partner-B">
+            <img src="./assets/images/daum.svg" alt="daumkakao" />
+            <img src="./assets/images/naver.svg" alt="naver" />
+          </div>
```

Comment:
- [ ] Great job on incorporating the logos for your partners section! However, I would like to suggest adjusting the size of the logos for desktop view, as they appear **a bit too large at the moment**. Resizing them **will ensure they are proportionate and visually appealing within the overall design**.👏

   ### Your design:
   ![image](https://github.com/alyconr/DevOpsConf2023/assets/107952724/4e35ac0a-49c9-48c7-a5a5-78e58f6b66f6)

   ### Behance design:
   ![image](https://github.com/alyconr/DevOpsConf2023/assets/107952724/c39e0560-9748-4848-920b-97b3e14695d5)